File: hackertracker/src/main/java/com/shortstack/hackertracker/Analytics/ItemEvent.java
package com.shortstack.hackertracker.Analytics;

import com.crashlytics.android.answers.CustomEvent;
import com.shortstack.hackertracker.Model.Item;

public class ItemEvent extends CustomEvent {

    public ItemEvent(AnalyticsController.Analytics event, Item item) {
        super(event.toString());
        putCustomAttribute("Id", item.getId());
        putCustomAttribute("Title", item.getTitle());
    }
}

File: hackertracker/src/main/java/com/shortstack/hackertracker/Analytics/ItemEvent.kt
package com.shortstack.hackertracker.analytics

import com.crashlytics.android.answers.CustomEvent
import com.shortstack.hackertracker.models.Item

class ItemEvent(event : AnalyticsController.Analytics, item : Item) : CustomEvent(event.toString()) {

    init {
        putCustomAttribute("Id", item.index)
        putCustomAttribute("Title", item.title)
    }
}
